src/client/fuse_ll.cc

```diff
@@ -163,6 +168,101 @@ class CephFuse::Handle {
   struct fuse_args args;
 };
 
+static bool already_mounted(const char *path)
+{
+  struct stat orig_stat;
+  struct stat parent_stat;
+  char *orig_path = NULL;
```

Comment:
```suggestion
  char path_copy[PATH_MAX];
```

No need for creating a new string. Use an assert to verify `strlen(path) < PATH_MAX`. Then you can do away with this `do { } while` loop.